Hud/Widgets/Colors/ColorGetter.cs

```diff
@@ -8,9 +8,9 @@ public static class ColorGetter
 {
     public static Vector4 GetVector(Color color) => color switch
```

Comment:
can we please also change the green and red to something more eye-hurting? like
        Color.Red => new Vector4(1, 0.2f, 0.2f, 1),
        Color.Green => new Vector4(0.58f, 0.85f, 0.46f, 1),